### PR TITLE
{bio}[] PLINK2 20210826

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by fizwit** [comment]
I am requesting that PLINK 2 be called PLINK2. PLINK 2 is a fork of PLINK and binary is called plink2.

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.6
See https://gist.github.com/3c07b6a500285cb552aa8a2a875768c9 for a full test report.

File: easybuild/easyconfigs/p/PLINK2/PLINK2-20210826-linux_x86_64.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+versionsuffix = '-linux_x86_64'
+
+homepage = 'https://www.cog-genomics.org/plink/2.0'
+description = """PLINK2 is a free, open-source whole genome association analysis toolset, designed to
+ perform a range of basic, large-scale analyses in a computationally efficient manner.
+ The focus of PLINK2 is purely on analysis of genotype/phenotype data."""
+
+toolchain = SYSTEM
+
+source_urls = ['http://s3.amazonaws.com/plink2-assets']
```
Comment (verdurin) [inline easybuild/easyconfigs/p/PLINK2/PLINK2-20210826-linux_x86_64.eb:18]: @fizwit seeing a `404` error when downloading the sources.

File: easybuild/easyconfigs/p/PLINK2/PLINK2-20210826-linux_x86_64.eb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# EasyBuild easyconfig
+#
+# Fred Hutch - Seattle WA US
+# John Dey jfdey@fredhutch.org
+easyblock = 'PackedBinary'
+
+name = 'PLINK2'
```
Comment (verdurin) [inline easybuild/easyconfigs/p/PLINK2/PLINK2-20210826-linux_x86_64.eb:7]: New name is fine by me.

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by fizwit** [comment]
Closing this PR because it is very out of date. PLINK2/20240205_linux_avx2 is available

**Pull request closed without merging**